app/controllers/request-to-go-live/index/get.controller.js

```diff
@@ -1,16 +1,82 @@
 'use strict'
 
+// NPM dependencies
+const lodash = require('lodash')
+
 // Local dependencies
 const goLiveStage = require('../../../models/go-live-stage')
 const response = require('../../../utils/response')
 
+const {
+  NOT_STARTED,
+  ENTERED_ORGANISATION_NAME,
+  CHOSEN_PSP_STRIPE,
+  CHOSEN_PSP_WORLDPAY,
+  CHOSEN_PSP_SMARTPAY,
+  CHOSEN_PSP_EPDQ,
+  TERMS_AGREED_STRIPE,
+  TERMS_AGREED_WORLDPAY,
+  TERMS_AGREED_SMARTPAY,
+  TERMS_AGREED_EPDQ,
+  LIVE,
+  DENIED
+} = goLiveStage
+
+const live = [
+  LIVE
+]
+
+const agreedToTerms = [
+  TERMS_AGREED_STRIPE,
+  TERMS_AGREED_WORLDPAY,
+  TERMS_AGREED_SMARTPAY,
+  TERMS_AGREED_EPDQ
+].concat(live)
```

Comment:
You can use `Destructuring assignment`, for example:
```
const agreedToTerms = [
  TERMS_AGREED_STRIPE,
  TERMS_AGREED_WORLDPAY,
  TERMS_AGREED_SMARTPAY,
  TERMS_AGREED_EPDQ,
  ...live
]
```

We use it in https://github.com/alphagov/pay-selfservice/blob/884eaf4eaf9b1b1796b65911f288110cef1f0959/app/routes.js#L174 and https://github.com/alphagov/pay-selfservice/blob/884eaf4eaf9b1b1796b65911f288110cef1f0959/app/utils/json_to_csv.js#L43 and I think it's the latest syntax which we discussed

Reply:
Ooh, I like that!